build_LGBM.232.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
```

Comment:
is this file necessary, or is it just left over from local development? It looks like it is leftover from local development and should be removed from this PR.

If this is the script to build this branch, it would be helpful to include this code in the pull request description.

Reply:
Right.  Some left over scripts from local development.  I'll remove them